### PR TITLE
Feat/success failure indicator

### PR DESCRIPTION
**Pull request by damien5701**

### CODE REVIEW

File: src/Components/Main.tsx [hunk 1]
```diff
@@ -90,6 +102,38 @@ export const Main = (props: IProps) => {
...
+            _warning++;
+            break;
+          default:
+            _failed++;
+            break;
+        }
+      }
+    });
+
+    _global = _success + _warning + _failed;
```
Comment (imranMnts) [inline src/Components/Main.tsx:129]: we should check if `_global != 0` to avoid any ArithmeticException

File: src/Components/Main.tsx [hunk 2]
```diff
@@ -90,6 +102,38 @@ export const Main = (props: IProps) => {
     if (loadingXLSX) onShare();
   }, [loadingXLSX]);
 
+  React.useEffect(() => {
+    let _global = 0;
```
Comment (imranMnts) [inline src/Components/Main.tsx]: can we add a new prop to disable this feature?

File: src/Components/Main.tsx (hunk 1 above)
Comment (damien5701) [reply src/Components/Main.tsx:129]: Done. I added a return if _global === 0

File: src/Components/Main.tsx (hunk 2 above)
Comment (damien5701) [reply src/Components/Main.tsx]: Done

File: src/index.tsx [hunk 3]
```diff
@@ -189,6 +190,7 @@ export const Netwatch: React.FC<IProps> = (props: IProps) => {
               nRequests={nRequests}
               connections={connections}
               clearAll={clearAll}
+              showStats={props.showStats ? true : false}
```
Comment (imranMnts) [inline src/index.tsx]: why not just `showStats={props.showStats}` ?

Comment (damien5701) [reply src/index.tsx]: I tried but I had a typescript error
Screen Shot 2021-03-29 at 11 41 06

Comment (damien5701) [reply src/index.tsx]: Fix done

**Review by imranMnts — APPROVED** [review]

**Pull request merged**